Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM theplant/jsonnet
+FROM public.ecr.aws/theplant/jsonnet:0.18.0
```

Comment:
Using a tag is good, but where is `0.18.0` coming from?

Reply:
I found the last time you mentioned it here https://github.com/theplant/plantbuild/pull/47#issuecomment-1101090456, so I copied both the `latest` and `0.18.0` tag from DockerHub to ECR.

Or we could just move it from our Folded repo to the upstream repo [public.ecr.aws/bitnami/jsonnet:0.18.0](https://gallery.ecr.aws/bitnami/jsonnet)?

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/18547234/204263844-886ef133-13ba-49d9-92f7-10d02f27d876.png">

<img width="1435" alt="image" src="https://user-images.githubusercontent.com/18547234/204263886-acc770ef-6c8e-42c1-bbd2-24d64747f553.png">

Reply:
Thank you, let's keep using versioned tags.

Reply:
From the comment you linked, we are using our own image because there's no official one from `google/jsonnet`, and `bitnami/jsonnet` is a 3rd party here rather than an upstream. I think that's still true?

Reply:
Yes, seems there is still no `google/jsonnet`

<img width="346" alt="image" src="https://user-images.githubusercontent.com/18547234/205192487-8a0a6764-19a7-40eb-bcff-3f295403a071.png">

Reply:
Thanks.

Let's keep using this base image to minimize the size of `theplant/plantbuild` for now.